registry/paraswap/calldata-AugustusSwapper-v5.json

```diff
@@ -0,0 +1,282 @@
+                        }
+                    }
+                ],
+                "required": ["factory", "initCode", "amountIn", "amountOutMin", "path"]
+            },
+            "0x935fb84b": {
+                "$id": "buyOnUniswap",
+                "fields": [
+                    {
+                        "path": "amountOut",
+                        "$ref": "$.display.definitions.amountOut"
+                    },
+                    {
+                        "path": "amountInMax",
+                        "$ref": "$.display.definitions.amountInMax"
+                    },
+                    {
+                        "path": "path.[]",
+                        "label": "Array of addresses",
+                        "format": "addressName",
+                        "params": {
+                            "types": ["wallet"]
+                        }
+                    }
+                ],
+                "required": ["amountOut", "amountInMax", "path"]
+            },
+            "0xc03786b0": {
+                "$id": "buyOnUniswapFork",
+                "fields": [
+                    {
+                        "path": "factory",
+                        "$ref": "$.display.definitions.factory"
+                    },
+                    {
+                        "path": "initCode",
+                        "format": "raw",
+                        "label": "init code"
+                    },
+                    {
+                        "path": "amountOut",
+                        "$ref": "$.display.definitions.amountOut"
+                    },
+                    {
+                        "path": "amountInMax",
+                        "$ref": "$.display.definitions.amountInMax"
+                    },
+                    {
+                        "path": "path.[]",
+                        "label": "Array of addresses",
+                        "format": "addressName",
+                        "params": {
+                            "types": ["wallet"]
+                        }
+                    }
+                ],
+                "required": ["factory", "initCode", "amountOut", "amountInMax", "path"]
+            },
+            "0x0b86a4c1" : {
+                "$id": "swapOnUniswapV2Fork",
+                "fields": [
+                    {
+                        "path": "tokenIn",
+                        "$ref": "$.display.definitions.tokenIn"
+                    },
+                    {
+                        "path": "amountIn",
+                        "$ref": "$.display.definitions.amountIn",
+                        "params": {
+                            "tokenPath": "tokenIn"
+                        }
+                    },
+                    {
+                        "path": "amountOutMin",
+                        "$ref": "$.display.definitions.amountOutMin"
+                    },
+                    {
+                        "path": "weth",
```

Comment:
This is the address of WETH ? I don't see the utility to display it in the clear signing review